playbooks/docker-build/docker-build.yaml

```diff
@@ -8,13 +8,13 @@
   register: dockerfile
 
 - name: Build a docker image
-  shell: "docker build -t wazopbx/$(basename $(pwd)) ."
+  shell: "{% raw %}docker image inspect --format='{{ .Id }}' wazopbx/$(basename $(pwd)) || docker build -t wazopbx/$(basename $(pwd)) .{% endraw %}"
```

Comment:
Not really useful to run inspect since the build doesn't use --no-cache, the build will be a noop

Reply:
not really from tests